src/heatmap/Heatmap.types.ts

```diff
@@ -0,0 +1,26 @@
+export interface HeatmapFilter {
+    label: string
+    backgroundColor?: string,
+    opacity?: number,
+    className?: string,
+    filterCondition: (item: any) => boolean
+}
+
+export type ContentNodeProps = {
+    tileElement: Record<string, any>
+}
+
+export interface HeatmapViewProps {
+    heatmapList: Array<Record<string, any>>
+    heatmapFilters?: HeatmapFilter[]
+    ContentNode: React.FC<ContentNodeProps>

```

Comment:
@devika-iouring  - ContentNode alone is given with a different naming conventions

cc: @revanth-iouring